dotnet/Makefile

```
@@ -129,6 +129,18 @@ LOCAL_WORKLOAD_TARGETS += Workloads/Microsoft.NET.Sdk.$(1)/LICENSE
 endef
 $(foreach platform,$(DOTNET_PLATFORMS),$(eval $(call WorkloadTargets,$(platform),$(shell echo $(platform) | tr A-Z a-z),$($(platform)_NUGET_VERSION_NO_METADATA))))
 
+$(DOTNET_NUPKG_DIR)/xamarin-workload.props: Workloads/xamarin-workload.template.props
+	$(Q) rm -f $@.tmp
+	$(Q_GEN) sed \
+		-e "s/@IOS_WORKLOAD_VERSION@/$(IOS_NUGET_VERSION).$(IOS_NUGET_COMMIT_DISTANCE)/g" \
+		-e "s/@MACCATALYST_WORKLOAD_VERSION@/$(MACCATALYST_NUGET_VERSION).$(MACCATALYST_NUGET_COMMIT_DISTANCE)/g" \
+		-e "s/@TVOS_WORKLOAD_VERSION@/$(TVOS_NUGET_VERSION).$(TVOS_NUGET_COMMIT_DISTANCE)/g" \
```

Something like this should work, and keep working when/if new platforms are added:

```suggestion
		$(foreach platform,$(DOTNET_PLATFORMS_UPPERCASE),-e "s/@$(platform)_WORKLOAD_VERSION@/$($(platform)_NUGET_VERSION).$($(platform)_NUGET_COMMIT_DISTANCE)/g)" \
```